### ISSUE TITLE
neus-facto quality worse than neus. Is this expected?

### ISSUE THREAD

**Issue by Akhila-ar** [autonomousvision/sdfstudio#126]
**Describe the bug**
I trained a neus (with hash encoding) and a neus-facto model on the same data (mipnerf-360 garden) with same iterations and same hash encoding network config. Then I extracted meshes at 512 resolution for both. I saw that the quality of mesh obtained from neus is much better than the neus-facto mesh. Is this the expected behaviour?

**To Reproduce**
Command I used to train neus:
`    
ns-train neus --pipeline.model.sdf-field.inside-outside True --experiment-name neus-garden \
        --trainer.steps_per_save 20000 --trainer.max_num_iterations 50001 --vis tensorboard \
        --pipeline.model.sdf-field.use_grid_feature True --pipeline.model.sdf-field.hidden-dim 256 --pipeline.model.sdf-field.num-layers 2 \
        --pipeline.model.sdf-field.num-layers-color 2 --pipeline.model.sdf-field.use-appearance-embedding False --pipeline.model.sdf-field.geometric-init True \
        --pipeline.model.sdf-field.bias 0.8 --pipeline.model.sdf-field.beta-init 0.3 --trainer.steps-per-eval-image 5000 --pipeline.datamanager.train-num-rays-per-batch 2048 \
        mipnerf360-data --data data/nerfstudio-data-mipnerf360/garden
`

Command I used to train neus-facto:
`    
ns-train neus-facto --pipeline.model.sdf-field.inside-outside True --experiment-name $exp_name \
        --pipeline.model.sdf-field.geometric-init True --pipeline.model.sdf-field.bias 0.8 \
        --trainer.max_num_iterations 50001 --vis tensorboard \
        mipnerf360-data --data data/nerfstudio-data-mipnerf360/garden
`

With these commands, I have tried to match the network configurations for the two models for a fair comparison. Is there anything I am missing here?

**Expected behavior**
I expected neus-facto to have better quality than neus. Is this expectation right?

**Screenshots**
neus output
<img width="651" alt="Screenshot 2023-06-08 at 6 02 46 PM" src="https://github.com/autonomousvision/sdfstudio/assets/89100852/cfc6bf45-3be5-4194-a442-ba519ab869bd">

neus-facto output
<img width="583" alt="Screenshot 2023-06-08 at 6 03 06 PM" src="https://github.com/autonomousvision/sdfstudio/assets/89100852/20e5d974-3f49-474e-a006-b1307cea4582">

**Additional context**

**Comment by niujinshuchong** [comment]
Hi, do you enable the background model in `neus-facto`? From your results, it seems like the background is also modeled with SDF?

**Comment by Akhila-ar** [comment]
No, I do not enable background model in neus-facto. As the default config of neus-facto in method_configs.py makes the background_model="none", I left that configuration to the default.

**Comment by Akhila-ar** [comment]
I did try enabling the background model by passing the flag `--pipeline.model.background-model grid` and `--pipeline.model.background-model mlp` . But I didnt see any improvement in the mesh quality. Attaching the results with background model below: 

background_model=grid
<img width="566" alt="Screenshot 2023-06-12 at 9 24 55 PM" src="https://github.com/autonomousvision/sdfstudio/assets/89100852/0d7f0bb0-e60d-4a5a-98ab-c92303a

background_model=mlp
<img width="552" alt="Screenshot 2023-06-12 at 9 25 17 PM" src="https://github.com/autonomousvision/sdfstudio/assets/89100852/5819cd9f-8f5e-4393-8e24-5d7c3412687c">
e0c0c">

**Comment by niujinshuchong** [comment]
Hi, sorry for the late reply. I think maybe the sphere bias (0.8) is too large at initialization and the sphere is treated as background quickly because hash-grids can learn the appearance very fast and this results in bad local minimal. 

And the near and far planes might not be set correctly when you use mipnerf360-data. I just updated the repo to use proposal network for background modeling. Could you try the latest commit?

**Comment by hturki** [comment]
I'm also trying to get neus-facto to work and this is what I get with the current commit on Garden in master:

<img width="2049" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/868618/d733ff71-5ede-4279-abe1-509193076b00">
<img width="2048" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/868618/457ce453-b55d-4ba8-9ef7-5ae93dfab4af">
<img width="2043" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/868618/5c8db1ad-7a7a-46ef-acde-856bdf4d6c0a">
